### PR TITLE
Added new resources in React Js ( Frontend )

### PR DESCRIPTION
**Pull request by mhdamaan79**
## Fixes Issue 

Closes (#706)

## Changes proposed
- [Add] React Official Documentation and Mosh's React tutorial

## Screenshots
LinksHub-React-resources

### CODE REVIEW

File: database/frontend/react.json [hunk 1]
```diff
@@ -6,5 +6,21 @@
...
+        "name":"React Tutorial for Beginners",
+        "description":"Mosh's React tutorial: beginner-friendly, well-structured, hands-on, builds solid React foundation.",
+        "url":"https://www.youtube.com/watch?v=SqcY0GlETPk",
+        "category":"frontend",
+        "subcategory":"react",
+        "language":"english"
+    },
+    {
+        "name":"React JS Official Documentation",
+        "description":"The go-to resource for React JS, offering comprehensive guides, tutorials, and API references to help you learn and master React development.",
```
Comment (CBID2) [inline database/frontend/react.json:20]: [Proofreading] This seems like this came from the website. Consider describing this source in your own words
```suggestion
        "description": "The go-to resource for React JS, offering comprehensive guides, tutorials, and API references to help you learn and master React development.",

```

File: database/frontend/react.json [hunk 2]
```diff
@@ -6,5 +6,21 @@
         "category":"frontend",
         "subcategory":"react",
         "language":"english"
+    },
+    {
+        "name":"React Tutorial for Beginners",
+        "description":"Mosh's React tutorial: beginner-friendly, well-structured, hands-on, builds solid React foundation.",
```
Comment (CBID2) [inline database/frontend/react.json:12]: [Proofreading]
```suggestion
        "description": This tutorial is beginner-friendly, well-structured, hands-on, and is helpful in understanding React",

```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @mhdamaan79! :) I pointed out some proofreading errors in my inline comments.

**Review by CBID2 — APPROVED** [review]
Great job @mhdamaan79! 😄 LGTM! 😄

**Review by CBID2 — APPROVED** [review]
Great job @mhdamaan79! 😄 LGTM! 😄

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @mhdamaan79! :) I noticed that some JSON errors popped up during the test run. Here's some information on it: https://github.com/rupali-codes/LinksHub/actions/runs/5072148109/jobs/9115441401?pr=711
Fix them and then ping me and @rupali-codes to review your pull request again.

**Comment by mhdamaan79** [comment]
> Hi @mhdamaan79! :) I noticed that some JSON errors popped up during the test run. Here's some information on it: https://github.com/rupali-codes/LinksHub/actions/runs/5072148109/jobs/9115441401?pr=711 Fix them and then ping me and @rupali-codes to review your pull request again.

Based on the changes you provided, I believe it's best to create a new pull request. Shall I proceed?

**Comment by mhdamaan79** [comment]
> Hi @mhdamaan79! :) I noticed that some JSON errors popped up during the test run. Here's some information on it: https://github.com/rupali-codes/LinksHub/actions/runs/5072148109/jobs/9115441401?pr=711 Fix them and then ping me and @rupali-codes to review your pull request again.

Take a look and let me know if anything else needs to be changed, Thanks! e2fa8c343acd9e97147589c2b25e1560d060a8ab
@CBID2 @rupali-codes

**Comment by mhdamaan79** [comment]
> > Hi @mhdamaan79! :) I noticed that some JSON errors popped up during the test run. Here's some information on it: https://github.com/rupali-codes/LinksHub/actions/runs/5072148109/jobs/9115441401?pr=711 Fix them and then ping me and @rupali-codes to review your pull request again.
> 
> Take a look and let me know if anything else needs to be changed, Thanks! [e2fa8c3](https://github.com/rupali-codes/LinksHub/commit/e2fa8c343acd9e97147589c2b25e1560d060a8ab) @CBID2 @rupali-codes

Kindly review the changes. Thank you! @CBID2

**Review by rupali-codes — APPROVED** [review]
lgtm

**Review by CBID2 — APPROVED** [review]
LGTM!

**Pull request merged**